TAIcheat/TAIcheat.cs

```diff
@@ -0,0 +1,695 @@
+﻿using OWML.Common;
+using OWML.ModHelper;
+using OWML.ModHelper.Events;
+using OWML.ModHelper.Input;
+using System.Reflection;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+using UnityEngine.SceneManagement;
+using System.Collections;
+using System;
+using System.Collections.Generic;
+using Harmony;
+
+namespace TAIcheat
```

Comment:
probably about time to remove this. PR is pretty close to merging :)

Reply:
Alright, I guess I can move EnableDebugMod to InputHandler instead as example

Reply:
done